### DISCUSSION TITLE
Pre-registration phase is now over!

### DISCUSSION THREAD

**Discussion by fpetric** [larics/icuas22_competition#35] [Announcements]
Dear Teams!

While we announced that the deadline for team lock-in and submission of enrollment proof documents will be ~~**March 15th**~~, there are quite a few more of you than we expected so we need to push this deadline a bit further back.

However, we will be closing the pre-registration form. If you filled it out, your team is in the system and you will be able to submit the additional documents. Teams that fail to provide the required paperwork will not be eligible for the finals.

As stated in the rulebook, you will need to **submit proof of enrollment in BSc, Btech, MSc, PhD or equivalent program for each team member, and the new deadline for this is** **April 15th** (the rulebook will be updated with this date and new version will be published on the website by the end of the week), so now is the time to start collecting the documents! The instructions where to submit the files will be sent to team leaders via e-mail, you can expect these e-mails to start arriving next week. 

Sidenote: we will be updating the evaluation procedure, we aim to make it more frequent. Information will be in the updated rulebook, and instructions for submission will be sent to team leaders (hopefully also next week).

**Comment by shu-bamma** [comment]
@fpetric can you please update this page ([link](http://www.uasconferences.com/2022_icuas/uav-competition-registered-teams/)) so that the teams can confirm our pre-registration for the competition.

**Reply by fpetric** [reply]
We are on it :)

**Reply by fpetric** [reply]
It should be updated now. Let us know if your team is not up there :)

**Reply by afaroo01** [reply]
@fpetric thanks for the update. So regarding the evaluation period, will it also start from April 15th or April 1st as informed earlier?

**Reply by fpetric** [reply]
We will try to start evaluating before April 1st. We are working on it :)
